<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_good</class>
 <widget class="QWidget" name="add_good">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление товара</string>
  </property>
  <widget class="QTableView" name="table_add_good">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>560</width>
     <height>180</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_new_data">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>231</y>
     <width>560</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>561</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_clear_data">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Очистить данные</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_add_good">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Добавить товар</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
